.circleci/config.yml

```diff
@@ -533,6 +561,8 @@ jobs:
       - checkout:
           path: *repo_path
       - setup-shared-env
+      - compile-all-jars:
+          restore_m2_cache: true
       - run-maven-with-pepper-options:
           phase: process-test-classes
           restore_m2_cache: true
```

Comment:
We only need to restore m2_cache once: it reads it back from wherever circle archives .m2/repository .

Reply:
Ah ha! The more you know! Will change it.